src/selectivity-base.js

```diff
@@ -181,6 +181,8 @@ function Selectivity(options) {
 
     this.setOptions(options);
 
+    this.$el.attr('tabindex', options.tabindex || 1);
```

Comment:
We could also get the tabindex from the select element. Then there's still the selectivity created without the select by giving the data as a parameter.

I'm also not too happy about forcing the default value as 1 since if there are multiple select fields then the tabindex will collide if non is given. 

Also given tabindex can collide with existing tabindexes.